test/integration/testsForApiVersion2.spec.js

```diff
@@ -0,0 +1,95 @@
+const { setupForApiVersion2 } = require('./util')
+const evrythng = require('../../dist/evrythng.node')
+
+describe('evrythng.js for apiVersion = 2', function () {
+  let settings = evrythng.setup({ apiVersion: 2, geolocation: false })
+  // await setupForApiVersion2(settings.apiUrl)
+
+  before(async () => {
+    settings = await evrythng.setup({ apiVersion: 2, geolocation: false })
+    await setupForApiVersion2(settings.apiUrl)
+  })
+
+  describe(`as Operator for ${settings.apiVersion}`, () => {
+    console.log(settings)
+    const scope = 'operator'
```

Comment:
Suggest `scopeType`, since this variable isn't actually a `Scope` object.

Reply:
fixed, thanks